go/grpc/pkg/greeter.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+option go_package = "gtihub.com/wso2/choreo-sample-apps/go/grpc-greeter/pkg";
+
+service greeter {
```

Comment:
```suggestion
service Greeter {
```

Reply:
Pb style guide docs suggest using PascalCase when defining service names. Shall we update it?

https://protobuf.dev/programming-guides/style/